tests/auto-selftest-example-test-pack/tests/empty_dir/subdir/example_with_no_tests.py

```diff
@@ -0,0 +1,2 @@
+# There should be no selftest file generated for this file because there are
+# no tests to be generated, but also the directory it's in should be deleted.
```

Comment:
Might also be worth having a test in a subdir that *does* have selftests generated for it?